source/leuven/tier2_hardware/superdome_hardware.rst

```diff
@@ -1,19 +1,20 @@
 Superdome hardware
 ==================
-Superdome is a shared memory machine and consists of several nodes.
-
-
-.. include:: tier2_login_nodes.rst
-
+Superdome is a single node with a large amount of memory (6 TB)
```

Comment:
I would have written it like:
"Superdome is a single-node machine equipped with 6 TB of memory, and is part of the Genius cluster."

Reply:
Indeed better like that (done in 34d94295f9fc4bbc8a30f23b80fdc2e510192267)